devicetypes/smartthings/zigbee-rgbw-bulb.src/zigbee-rgbw-bulb.groovy

```diff
@@ -213,7 +213,11 @@ def setGenericName(value){
 }
 
 def setLevel(value, rate = null) {
-	zigbee.setLevel(value)
+	if (value == 0) {
```

Comment:
We actually take care of this inside `zigbee.setLevel` so you don't need to do it here

Reply:
I hurried up with PR:)